content/docs/hello-world.md

```diff
@@ -6,7 +6,7 @@ prev: cdn-links.html
 next: introducing-jsx.html
 ---
 
-The smallest React example looks like this:
+Het meest eenvoudige React-voorbeeld ziet er zo uit:
```

Comment:
@Keraito Dutch has awful rules concerning "samenstellingen" with foreign words, which is essentially what we're trying to form here. Personally I think the hyphen is needed to be 100% grammatically correct, but it might be better for readability to mitigate it.